apps/wappush/test/unit/wappush_test.js

```diff
@@ -793,6 +825,10 @@ suite('WAP Push', function() {
       this.sinon.spy(window, 'close');
     });
 
+    teardown(function() {
+      clock.restore();
+    });
```

Comment:
nit: you don't need this because we use  `this.sinon.useFakeTimers()` -- it's automatically restored at teardown.